bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -687,6 +687,20 @@ private function addObjectsNode(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->booleanNode('allow_processing_unpublished_elements_for_url_slug')
```

Comment:
This is now the wrong location for the config, since it also covers static routes. 
I'd propose the following location/naming: `pimcore.routing.allow_processing_unpublished_fallback_document`